src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.html

```diff
@@ -1,84 +1,181 @@
 <div class="builder-navbar-wrapper">
   <div class="top">
     <div class="top__left">
-      <a *ngIf="!adminMode" routerLink="onion/dashboard"><i class="far fa-arrow-left"></i> Back to Dashboard</a>
+      <a *ngIf="!adminMode" routerLink="onion/dashboard"
+        ><i class="far fa-arrow-left"></i> Back to Dashboard</a
+      >
       <!-- FIXME: Convert to routerLink when admin dashboard is pulled into CLARK -->
-      <a *ngIf="adminMode" href="https://admin.clark.center"><i class="far fa-arrow-left"></i> Back to Dashboard</a>
+      <a *ngIf="adminMode" href="https://admin.clark.center"
+        ><i class="far fa-arrow-left"></i> Back to Dashboard</a
+      >
     </div>
     <div class="top__right">
-      <div *ngIf="!['unreleased', 'rejected'].includes(learningObject?.status) && learningObject?.collection" class="top__collection"
-        tip="This learning object has been submitted to the {{ collection.name }} collection." tipLocation="bottom">
+      <div
+        *ngIf="
+          !['unreleased', 'rejected'].includes(learningObject?.status) &&
+          learningObject?.collection
+        "
+        class="top__collection"
+        tip="This learning object has been submitted to the {{
+          collection.name
+        }} collection."
+        tipLocation="bottom"
+      >
         <span>{{ collection.name }}</span>
-        <div class="collection__image" [style.background-image]="'url(assets/images/collections/' + learningObject?.collection + '.png)'"></div>
+        <div *ngIf="collection.hasLogo; else genericCollectionLogoTemplate"
```

Comment:
Most of the changes in this file are due to formatting. This is an actual change.